src/components/modal/ModalDialog.js

```diff
@@ -6,6 +6,7 @@ import uniqueId from 'lodash/uniqueId';
 import { defineMessages, injectIntl } from 'react-intl';
 
 import IconClose from '../../icon/fill/X16';
+import IconChevron from '../../icons/general/IconChevron';
```

Comment:
I believe these icons have been deprecated. The closest thing I see in `icon/fill` is the Circle Back icon.